src/Modal/Content.js

```diff
@@ -0,0 +1,27 @@
+import React, { Fragment } from 'react'
+import propTypes from 'prop-types'
+import css from 'styled-jsx/css'
+import cx from 'classnames'
+
+export const Content = ({ children, scrollable }) => (
+    <div className={cx('content', { scrollable })}>
+        {children}
+
+        <style jsx>{`
+            .content {
```

Comment:
Same situation here, you can easily target `div` over `.content` for extra isolation.